coding-concepts.js

```diff
@@ -7,48 +7,58 @@
 
 // --------------------INSTRUCTOR EXAMPLE: What will this log?
 const colors = ["tangerine", "magenta", "lilac", "daffodil"]
-// console.log(colors.push("indigo"))
+console.log(colors.push("indigo"))
+
+// a) Your answer: 5
+// b) Verify and explain: 
+//The example indicates that the variables in this string  array with a built-in method data type index will output a total number value of the words in parentheses separated by the comma after "indigo" is added to the end. The total word characters in the index is "4" and when you add the color "indigo" it will equal to "5".
 
-// a) Your answer:
-// b) Verify and explain:
 
 // --------------------1) What will this log?
 
 const cohort = "LEARN 2023"
-// console.log(cohort.length)
+console.log(cohort.length)
 
-// a) Your answer:
+// a) Your answer: 10
 // b) Verify and explain:
+//The example is a variable string array data type that asks for the total length and will count the number value of characters in the parentheses including the empty space. The total number of characters equals ten including the space in between "N" and "2". 
```

Comment:
Nice job! 👏🏽 Im glad you counted the space. Just keep in mind that an array would be signaled by [] , here we have just a string.